xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -1845,7 +1845,7 @@ bool CVideoPlayer::GetCachingTimes(double& level, double& delay, double& offset)
 
   delay  = 0.0;
   level  = 0.0;
-  offset = (double)(cached + queued) / length;
+  offset = (cached + queued) / length;
```

Comment:
not sure about this. cached is of type uint64_t. means all operation will be done to this type?

Reply:
uint64_t + double = double